### DISCUSSION TITLE
Does Ray support Lumen?

### DISCUSSION THREAD

**Discussion by mennovanhout** [spatie/laravel-ray#70] [Q&A]
Hi there,

I could test this but it may be a little bit easier to just ask. Is ray also tested on Lumen? with the laravel package?

**Comment by freekmurze** [comment]
Yes, there is support for Lumen

**Reply by mennovanhout** [reply]
Aweomse then I'll install it right away.

**Comment by mennovanhout** [comment]
Doesn't seem to work i'll make an issue

spatie/laravel-ray#71
